src/main/java/com/devonfw/quarkus/general/service/exception/mapper/UnauthorizedExceptionMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.devonfw.quarkus.general.service.exception.mapper;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import io.quarkus.security.UnauthorizedException;
+
+@Provider
+public class UnauthorizedExceptionMapper extends AbstractExceptionMapper
+    implements ExceptionMapper<UnauthorizedException> {
```

Comment:
@GuentherJulian why do we need this mappers?

Reply:
@maybeec to have a proper exception handling with consistent error messages throughout the services like we have it in devon4j with the [RestServiceExceptionFacade](https://github.com/devonfw/devon4j/blob/master/modules/rest/src/main/java/com/devonfw/module/rest/service/impl/RestServiceExceptionFacade.java). The default mapper provided by RESTEasy do not give that much information (usually only status and maybe a simple text).